.github/workflows/kbs_build_and_push.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build and Push kbs Image
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Get Commit SHA
+      id: commit_sha
+      run: echo "::set-output name=sha::$(git rev-parse --short ${{ github.sha }})"
+
+    - name: Login to quay Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+
+    - name: Build Container Image
+      run: |
+        commit_sha=${{ steps.commit_sha.outputs.sha }}
+
+        DOCKER_BUILDKIT=1 docker build -t quay.io/confidential-containers/kbs:coco-as-${commit_sha} . -f docker/Dockerfile --push; \
+        DOCKER_BUILDKIT=1 docker build -t quay.io/confidential-containers/kbs:coco-as-openssl-${commit_sha} --build-arg KBS_FEATURES=coco-as-builtin,openssl,resource,opa . -f docker/Dockerfile --push; \
+        DOCKER_BUILDKIT=1 docker build -t quay.io/confidential-containers/kbs:coco-as-grpc-${commit_sha} . -f docker/Dockerfile.coco-as-grpc --push
```

Comment:
Can we limit it to a single configuration that is bundles most common features first? 

```yaml
- name Build and Push Container Image:
  run: DOCKER_BUILDKIT=1 docker build -t ghcr.io/confidential-containers/kbs-${{ github.sha }} --build-arg KBS_FEATURES=coco-as-builtin,openssl,resource,opa . -f docker/Dockerfile --push
```